locales-en-US.xml

```diff
@@ -275,6 +447,58 @@
     </term>
 
     <!-- SHORT ROLE FORMS -->
+    <term name="chair">
```

Comment:
@adam3smith @bdarcus I could use a hand here. APA doesn't use short forms for any of these media roles. Do other fields?

Reply:
I think APA may be the only style that uses chair in citations. I'd just set the short form to "chair" as well (I've seen chr. as an acronym, but I don't think it's particularly common or useful).

Reply:
@adam3smith Could you help out with compiler/curator/producer/etc. below?